test/tools/OpenCover/OpenCover.psm1

```diff
@@ -519,6 +520,9 @@ function Invoke-OpenCover
             # poll for process exit every 60 seconds
             # timeout of 6 hours
             $timeOut = ([datetime]::Now).AddHours(6)
```

Comment:
I think it would be worth putting that data in the comment to give context to 6 hours